tests/cases/fourslash/organizeImports8.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path="fourslash.ts" />
+
+////import { foo as foo } from "foo";
+////foo;
+
+verify.organizeImports(
+`import { foo } from "foo";
+foo;`
+);

```

Comment:
Maybe add another test that includes dropping imports, and consolidating multiple renames.

```ts
/// <reference path="fourslash.ts" />

////import { foo as foo, bar, baz, bizzle as bizzle, frizzle as frizzle } from "mod";
////foo(bar, frizzle);

verify.organizeImports(
`import { foo, bar, frizzle } from "mod";
foo(bar, frizzle);`
);
```

Reply:
I've added test